pkg/query-service/app/integrations/manager.go

```diff
@@ -4,60 +4,111 @@ import (
 	"context"
 	"fmt"
 	"slices"
+	"strings"
 	"time"
 
+	"github.com/jmoiron/sqlx"
+	"go.signoz.io/signoz/pkg/query-service/app/dashboards"
 	"go.signoz.io/signoz/pkg/query-service/app/logparsingpipeline"
 	"go.signoz.io/signoz/pkg/query-service/model"
 )
 
 type IntegrationAuthor struct {
-	Name     string
-	Email    string
-	HomePage string
+	Name     string `json:"name"`
+	Email    string `json:"email"`
+	HomePage string `json:"homepage"`
 }
 type IntegrationSummary struct {
-	Id          string
-	Title       string
-	Description string // A short description
+	Id          string `json:"id"`
+	Title       string `json:"title"`
+	Description string `json:"description"` // A short description
 
-	Author IntegrationAuthor
+	Author IntegrationAuthor `json:"author"`
+
+	Icon string `json:"icon"`
 }
 
 type IntegrationAssets struct {
-	// Each integration is expected to specify all log transformations
-	// in a single pipeline with a source based filter
-	LogPipeline *logparsingpipeline.PostablePipeline
+	Logs       LogsAssets             `json:"logs"`
+	Dashboards []dashboards.Dashboard `json:"dashboards"`
+
+	// TODO(Raj): Maybe use a struct for alerts
+	Alerts []map[string]interface{} `json:"alerts"`
```

Comment:
Please use this struct https://github.com/SigNoz/signoz/blob/3fece44aef9375538a2e19ecc1186d5499c10bf4/pkg/query-service/rules/apiParams.go#L34-L58